merino/providers/weather/backends/accuweather.py

```diff
@@ -92,18 +94,26 @@ async def get_weather_report(
         async with AsyncClient(base_url=self.url_base) as client:
             if not (location := await self.get_location(client, country, postal_code)):
                 return None
+            try:
+                async with asyncio.TaskGroup() as tg:
+                    task_current = tg.create_task(
+                        self.get_current_conditions(client, location.key)
+                    )
+                    task_forecast = tg.create_task(
+                        self.get_forecast(client, location.key)
+                    )
+            except BaseExceptionGroup as e:
```

Comment:
`ExceptionGroup` should be sufficient here.
```suggestion
            except ExceptionGroup as e:
```